source/common/router/upstream_request.cc

```diff
@@ -57,7 +57,7 @@ UpstreamRequest::UpstreamRequest(RouterFilterInterface& parent,
   if (parent_.config().start_child_span_) {
     span_ = parent_.callbacks()->activeSpan().spawnChild(
         parent_.callbacks()->tracingConfig(), "router " + parent.cluster()->name() + " egress",
-        parent.timeSource().systemTime());
+        parent_.callbacks()->streamInfo());
```

Comment:
what makes  `parent.timeSource().systemTime` here interchangable with `stream_info.startTime()`?

Reply:
child span may take not only system time but also stream-related info in Apache SkyWalking.

Reply:
> I thinks the this replace may results in unaccurate start time for all the child span from spawnChild.

Ah I missed it... We need revisit about it.